python/servo/package_commands.py

```diff
@@ -0,0 +1,135 @@
+    @CommandArgument('--dev', '-d', action='store_true',
+                     help='Package the dev build')
+    @CommandArgument('--android',
+                     default=None,
+                     action='store_true',
+                     help='Package Android')
+    def package(self, release=False, dev=False, android=None, debug=False, debugger=None):
+        env = self.build_env()
+        if android is None:
+            android = self.config["build"]["android"]
+        binary_path = self.get_binary_path(release, dev, android=android)
+        if android:
+            if dev:
+                env["NDK_DEBUG"] = "1"
+                env["ANT_FLAVOR"] = "debug"
+                dev_flag = "-d"
+            else:
+                env["ANT_FLAVOR"] = "release"
+                dev_flag = ""
+
+            target_dir = os.path.dirname(binary_path)
+            output_apk = "{}.apk".format(binary_path)
+            try:
+                with cd(path.join("support", "android", "build-apk")):
+                    subprocess.check_call(["cargo", "run", "--", dev_flag, "-o", output_apk, "-t", target_dir,
+                                           "-r", self.get_top_dir()], env=env)
+            except subprocess.CalledProcessError as e:
+                print("Packaging Android exited with return value %d" % e.returncode)
+                return e.returncode
+        else:
+            dir_to_package = '/'.join(binary_path.split('/')[:-1])
+            browserhtml_path = find_dep_path_newest('browserhtml', binary_path)
+            if browserhtml_path is None:
+                print("Could not find browserhtml package; perhaps you haven't built Servo.")
+                return 1
+            print("Deleting unused files")
+            keep = ['servo', 'resources', 'build']
+            for f in os.listdir(dir_to_package + '/'):
+                if f not in keep:
+                    delete(dir_to_package + '/' + f)
+            for f in os.listdir(dir_to_package + '/build/'):
+                if 'browserhtml' not in f:
+                    delete(dir_to_package + '/build/' + f)
+            print("Writing runservo.sh")
+            # TODO: deduplicate this arg list from post_build_commands
+            servo_args = ['-w', '-b',
+                          '--pref', 'dom.mozbrowser.enabled',
+                          '--pref', 'dom.forcetouch.enabled',
+                          '--pref', 'shell.quit-on-escape.enabled=false',
+                          path.join(browserhtml_path, 'out', 'index.html')]
+
+            runservo = os.open(dir_to_package + 'runservo.sh', os.O_WRONLY | os.O_CREAT, int("0755", 8))
+            os.write(runservo, "./servo " + ' '.join(servo_args))
+            os.close(runservo)
+            print("Creating tarball")
+            tar_path = '/'.join(dir_to_package.split('/')[:-1]) + '/'
+            tar_path += time.strftime('%Y-%m-%d-%H%M') + "-servo-tech-demo.tar.gz"
```

Comment:
Can we please use ISO 8601 format?

```
import datetime

datetime.utcnow().replace(microsecond=0).isoformat()
```

Ideally this would also indicate that this is UTC time in the printed representation, but Python 2 doesn't have `datetime.timezone.utc`, and I don't think an extra dependency or writing a UTC `tzinfo` subclass is worth it..